mods/d2k/rules/infantry.yaml

```diff
@@ -41,8 +41,9 @@ engineer:
 	Mobile:
 		Speed: 31
 	EngineerRepair:
+	CaptureManager:
 	Captures:
-		CaptureTypes: building, husk
+		CaptureTypes: building
```

Comment:
This probably wants to be reverted again (as discussed on IRC).

Reply:
As discussed in IRC, this was leftover from #9615 which removed the capturing.